csharp/ReadJsonFile.cs

```diff
@@ -0,0 +1,27 @@
+// Read JSON file, transform and print to stdout
+using System;
+using System.IO;
+using System.Text.Json;
+
+public class ReadJsonFile{
+    public static void Main(string[] args){
+        string jsonFilePath = args[0];
+        string jsonContent = File.ReadAllText(jsonFilePath);
+
+        JsonDocument doc = JsonDocument.Parse(jsonContent);
+        JsonElement root = doc.RootElement;
+
+        foreach (JsonElement person in root.EnumerateArray()){
+            if (person.ValueKind == JsonValueKind.Object){
```

Comment:
Let's not check to see if `person.ValueKind == JsonValueKind.Object`. Instead, we can just use `person` and it will raise an exception of it's not an `Object`